package/cfg/config.yaml

```diff
@@ -211,25 +211,13 @@ managedservices:
 
 # TODO: Clean up in the next refactor
 version_mapping:
-  "rke-1.13":
-    - "rke-cis-1.4"
-  "rke-1.14":
-    - "rke-cis-1.4"
-  "rke-1.15":
-    - "rke-cis-1.5-permissive"
-  "rke-1.16":
-    - "rke-cis-1.6-permissive"
-  "rke-1.17":
-    - "rke-cis-1.6-permissive"
-  "rke-1.18":
-    - "rke-cis-1.6-permissive"
-  "rke-1.19":
-    - "rke-cis-1.20-permissive"
-  "rke-1.20":
-    - "rke-cis-1.20-permissive"
-  "v1.18.10+rke2r1":
-    - "rke2-cis-1.5-hardened"
-    - "rke2-cis-1.5-permissive"
+  "1.23": "cis-1.23"
+  "1.24": "cis-1.24"
+  "1.25": "cis-1.7"
+  "1.26": "cis-1.8"
+  "1.27": "cis-1.9"
+  "1.28": "cis-1.9"
```

Comment:
Should this not go under `# CIS - Generic profiles`?

Reply:
Thanks @pjbgf ! This part concerns the `version_mapping`, which tells kube-bench to use a specific version for a given k8s version, if no profile argument is passed.
We reorganized the target_mapping by k8s distros and profile types a while ago as it was very messy, and it's where we commented with `# CIS - Generic profiles`,  `# rke1: Generic`, etc.
IMO we could improve the comments for the version_mapping par to mention what this section is used for.

Reply:
Thanks for clarifying, that makes sense. :bow: